.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -14,3 +14,6 @@ Give us a quick code demo, command, or another example that couldn't be solved w
 
 **Additional context**
 Add any other context about the request here!
+
+**How does this increase our chances of an iCrawl Acquisition?**
+Please explain how this will increase our relative value to iCrawl in order to further our goal of acquisition
```

Comment:
```suggestion
**How does this increase our chances of an iCrawl Acquisition?**
Please explain how this will increase our relative value to iCrawl in order to further complete our final goal and life purpose of the acquisition
```

Reply:
LOL